enterprise/server/bes_artifact_uploader/bes_artifact_uploader.go

```diff
@@ -0,0 +1,104 @@
+package bes_artifact_uploader
```

Comment:
could this be `package bes`?

Then your code would read bes.Uploader (and would not stutter)

Reply:
Renamed to `bes_artifacts` to avoid making the scope of this package seem too broad; wdyt?